amazon/iontest/ion_test_driver.py

```diff
@@ -26,6 +26,8 @@
 
     --git <path>                        Path to the git executable.
 
+    --maven <path>                      Path to the maven executable.
```

Comment:
We probably also need one for `java`.

Reply:
Sure, what's java one's command? (e.g `javac`, `java` or `java -jar`)

Reply:
`java`

Reply:
Added. In case I missed anything, are we going to use `java` somewhere or just keep it for future use?

Reply:
It's already used. `java -jar`. That reminds me, we should use the specified path if provided. So the first element of the prefix for the ion-java executable should be `TOOL_DEPENDENCIES['java']` instead of just `java`.

Reply:
Alright, changed.